CHANGES.rst

```diff
@@ -1,14 +1,16 @@
 Changes
 *******
 
-0.12.0 (unreleased)
+0.11.2 (2023-07-17)
```

Comment:
Wrong date.  This should be updated only on the day this PR is merged.

Reply:
Huh, we went from `0.11.0` to `0.11.2` in the change log?

Reply:
We already tagged a version as `0.11.1`, but it has no release notes.

Reply:
Release date for 0.11.2 still wrong?  Unless the tag already pushed before this PR merged?

Reply:
I figured that I'll update the date of release just before we merge.,